yourssu-assignment-1/yourssu-assignment-1/ButtonView.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  ButtonView.swift
+//  yourssu-assignment-1
+//
+//  Created by 박윤빈 on 2023/09/30.
+//
+
+import SwiftUI
+
+struct ButtonView: View {
+    
+    @Binding var firstInput: String
+    @Binding var secondInput: String
+    @Binding var resultText: String
+    
+    var body: some View {
+        VStack(spacing: 10) {
+            Button {
```

Comment:
Button 전반적인 부분에 해당하는 건데요! 버튼을 보면 사실 함수랑 Label에 Text제외하곤 나머지가 동일한데 ButtonStyle로 Custom Button 하나를 만들어서 간결하게 줄여보는 건 어떨까요?
[ButtonStyle에 관한  애플 문서 링크](https://developer.apple.com/documentation/swiftui/buttonstyle)
[ButtonStyle 예시 (velog)](https://velog.io/@comdongsam/SwiftUI-ButtonStyle)

Reply:
```suggestion
struct capsuleText: View {
    var text: String
    
    var body: some View {
        Text(text)
            .font(.largeTitle)
            .padding()
            .background(Color.blue)
            .clipShape(Capsule())
    }
}
 
struct ContentView: View {
    
    var body: some View {
        VStack {
            capsuleText(text: "First")
                .foregroundColor(.white)
            capsuleText(text: "Second")
                .foregroundColor(.yellow)
        }
    }
    
}
```

이런식으로 따로 struct로 빼는 방법도 있어요! 다양한 방법을 시도해봐요!